README.rst

```diff
@@ -60,10 +60,9 @@ Usage Example
 
     import time
     import board
-    import busio
     import adafruit_hts221
 
-    i2c = busio.I2C(board.SCL, board.SDA)
+    ii2c = board.I2C()
```

Comment:
typo

Reply:
Thanks

Reply:
@ladyada changes in https://github.com/adafruit/Adafruit_CircuitPython_HTS221/pull/8/commits/cd3be8c39faacc6f71ada72a70a8efaa8ef2c1fc. Thanks